guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
<p>Once the model is trained, we need to visualize the results and measure the performance. </p><ul><li><code>show_results()</code>&nbsp;- The training results can be printed using <code>show_results()</code> method to assess the quality of the trained model and compute the model's metrics. The&nbsp;<code>show_results()</code>&nbsp;method will typically output a table where the last column will contain the values predicted by the model after being applied to the validation dataset. Comparing the predicted values with the actual target variable will give some indication of the performance of the trained model.</li><li><br></li><li><code>score()</code>&nbsp;- Subsequently, the model metrics of the trained model can be estimated using the&nbsp;<code>score()</code>&nbsp;function. This function currently returns the r-square/accuracy, which also reflects model performance.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>